src/libraries/System.Data.Common/ref/System.Data.Common.cs

```diff
@@ -694,50 +729,52 @@ public partial class DataTable : System.ComponentModel.MarshalByValueComponent,
         protected virtual void OnTableCleared(System.Data.DataTableClearEventArgs e) { }
         protected virtual void OnTableClearing(System.Data.DataTableClearEventArgs e) { }
         protected virtual void OnTableNewRow(System.Data.DataTableNewRowEventArgs e) { }
-        public System.Data.XmlReadMode ReadXml(System.IO.Stream stream) { throw null; }
-        public System.Data.XmlReadMode ReadXml(System.IO.TextReader reader) { throw null; }
+        public System.Data.XmlReadMode ReadXml(System.IO.Stream? stream) { throw null; }
+        public System.Data.XmlReadMode ReadXml(System.IO.TextReader? reader) { throw null; }
         public System.Data.XmlReadMode ReadXml(string fileName) { throw null; }
-        public System.Data.XmlReadMode ReadXml(System.Xml.XmlReader reader) { throw null; }
-        public void ReadXmlSchema(System.IO.Stream stream) { }
-        public void ReadXmlSchema(System.IO.TextReader reader) { }
+        public System.Data.XmlReadMode ReadXml(System.Xml.XmlReader? reader) { throw null; }
+        public void ReadXmlSchema(System.IO.Stream? stream) { }
```

Comment:
It's really strange that these all allow null, but, yeah, I see that in the implementation. :)

Reply:
Agreed... A lot of these APIs are inappropriately lax about nulls.